src/ScriptTools.sol

```diff
@@ -63,7 +63,10 @@ library ScriptTools {
      * @return config The raw json text of the config.
      */
     function loadConfig(string memory name) internal returns (string memory config) {
-        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", readInput(vm.envOr("FOUNDRY_SCRIPT_CONFIG", name)));
+        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", string(""));
+        if (eq(config, "")) {
+            config = readInput(vm.envOr("FOUNDRY_SCRIPT_CONFIG", name));
+        }
```

Comment:
What exactly fixes it?
Is it that with the prev way doing `export FOUNDRY_SCRIPT_CONFIG_TEXT = ""` was making to enter the path like it wouldn't be empty?

Reply:
The problem is that it if `FOUNDRY_SCRIPT_CONFIG_TEXT` is set and the script config or default name file doesn't exist it will error. This is not an optimistic OR operation.

Reply:
Ohh ok got it.